cmd/kubeadm/app/componentconfigs/kubelet_windows.go

```diff
@@ -0,0 +1,79 @@
+//go:build windows
+// +build windows
```

Comment:
i think build tags are not needed given the _windows suffix in the file name?

Reply:
Yes, you are right. This can be omitted, but it seems that our codebase always explicitly declares these build tags.

Such as: https://github.com/kubernetes/kubernetes/blob/c7d57a01b7291c19d3fdd51fd1cccff5dd4787f9/pkg/volume/util/fs/fs_windows.go#L1-L2

https://github.com/kubernetes/kubernetes/blob/c7d57a01b7291c19d3fdd51fd1cccff5dd4787f9/pkg/kubelet/cadvisor/cadvisor_windows.go#L1-L2

Reply:
Those tags in the kubelet can be cleaned too i guess.

Reply:
Let's just remove these build tags.